### ISSUE TITLE
Split explicit time integration challenges

### ISSUE THREAD

**Issue by pwolfram** [MPAS-Dev/MPAS-Model#217]
Following #206, tested split-explicit time stepping was not able to provide a boundary condition with a reasonable time scale for relaxation to the coastal boundary, unlike RK4, which suggests the time integrator will not be suitable for application of tidally-forced boundary conditions via use of restoring over a short time scale. 

cc @mark-petersen

**Comment by xylar** [comment]
@pwolfram, could you rephrase this is more of an issue to be solved?  First, the title is so broad that I wouldn't know what it is useful for just scrolling through the MPAS-Model issues.  Second, I know that restoring at boundaries does work in certain conditions with split-explicit because I use boundary restoring in ISOMIP+ test cases.  I think issues should be phrased as something that could be addressed with a pull request (or at most a few), rather than either a place to list challenges one is facing or an open-ended and vague feature request.

**Comment by pwolfram** [comment]
@xylar, the issue is related to the performance of the split explicit time integrator, as I've discussed with @mark-petersen.  

In #206, the tau restoring timescale needs to be roughly an order of magnitude larger for split explicit than for RK4 (just for stability), which means that the tidal restoring boundary condition effectively does not work with split explicit.  @mark-petersen has mentioned that time-step based convergence of the integrator is also not as expected, although I have not explored this personally.  

Thus, following improvements to split explicit, #206 can be used as one of the test beds or metrics to make sure its performance is improved by switching the time integrator from 'RK4' to 'split_explicit' (which is unstable if you were to try this out now).

**Comment by xylar** [comment]
Great, thanks you for clarifying the title and the scope of the issue.

**Comment by pwolfram** [comment]
Thanks for asking a key question to clarify @xylar!
